### PR TITLE
Update Compatibility.cs for Lenovo GeekPro G5000

### PR DESCRIPTION
**Pull request by TuanZiGit**
I check the compatibility for GeekPro G5000 and it's okay so I added it

### CODE REVIEW

File: LenovoLegionToolkit.Lib/Utils/Compatibility.cs [hunk 1]
```diff
@@ -257,4 +260,4 @@ private static bool IsBiosVersionMatch(string currentBiosVersionString, (string,
 
         return false;
     }
-}
```
Comment (TuanZiGit) [inline LenovoLegionToolkit.Lib/Utils/Compatibility.cs:260]: idk why there's "- }" and "+ {"

Comment (BartoszCichecki) [reply LenovoLegionToolkit.Lib/Utils/Compatibility.cs:260]: Doesn't matter, don't worry.

**Review by BartoszCichecki — APPROVED** [review]

**Comment by TuanZiGit** [comment]
So how can I remove the "LOGGING ENABLED" for my device?

**Comment by TuanZiGit** [comment]
I tested it but it says not supported
I reviewed the prefix of the model and it is correct: "82XV"

**Comment by BartoszCichecki** [comment]
Please screenshot the unsupported windows and paste it here.

**Comment by TuanZiGit** [comment]
image
I'm using the installer generated by Github Actions yesterday.

**Pull request closed without merging**